sdk/core/Azure.Core/tests/Azure.Core.Tests.csproj

```diff
@@ -2,9 +2,14 @@
   <PropertyGroup>
     <ProjectGuid>{84491222-6C36-4FA7-BBAE-1FA804129151}</ProjectGuid>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <DefineConstants>$(DefineConstants);TESTFRAMEWORK;HAS_INTERNALS_VISIBLE_CORE</DefineConstants>
+    <DefineConstants>$(DefineConstants);HAS_INTERNALS_VISIBLE_CORE</DefineConstants>
```

Comment:
Any reason not to use a Shared Source project? See sdk\keyvault. This is supported by Visual Studio seamlessly. Or is just because this can be used to also build the project?

Reply:
Yes, I think that is the idea - the project can be built and possibly shipped in the future.